Modules/Libs/QuestieLib.lua

```diff
@@ -338,14 +338,19 @@ end
 local cachedTitle = nil
 -- Move to Questie.lua after QuestieOptions move.
 function QuestieLib:GetAddonVersionInfo()
+    local name = "Questie"
+    local version = GetAddOnMetadata(name, "Version")
+    -- %d = digit, %p = punctuation character, %x = hexadecimal digits.
+    local major, minor, patch = string.match(version, "(%d+)%p(%d+)%p(%d+)")
+
     if (not cachedTitle) then
+        --[[ not really sure why QuestiDev-master is checked
```

Comment:
This is because the TOC and the repo folder is named this way. Not ideal and might be changed at some point. Currently we need this.